reactDashboard/src/App.css

```diff
@@ -86,6 +102,22 @@ button.MuiButtonBase-root {
   font-size: 0.75rem;
 }
 
+.darkmode button.MuiButtonBase-root {
+  background-color: inherit;
+  color: #f2f2f2;
+  border-color: #f2f2f2;
+}
+
 button.MuiButtonBase-root:hover {
-  box-shadow: 5px 5px 5px #aaaaaa;
+  box-shadow: none;
+  background-color: #f2f2f2;
+  color: rgb(56, 56, 56)
+}
+
+.darkmode .container-fluid {
+  background-color: #121212;
+}
+
+.darkmode div {
```

Comment:
That seems to be a very generic and risky declaration - did you mean all divs in the app?

Reply:
@donal-s it's gone in my next push